docker-compose.yml

```diff
@@ -0,0 +1,73 @@
+version: '3.2'
```

Comment:
Once we move the docker-compose to the top directory, we might need add some contents to the top [README](https://github.com/confidential-containers/kbs/blob/main/README.md) to help the users who is not familiar with the project to use this? like a link to https://github.com/confidential-containers/kbs/blob/main/kbs/docs/cluster.md

wdyt?

Reply:
I have a number of changes to the docs planned, but I was thinking I would do that in a follow-up PR, especially so we can get the fix in this PR merged quickly. Does that work for you?

Reply:
It works

Reply:
See https://github.com/confidential-containers/kbs/pull/287